Specification/BrAPI-Schema/BrAPI-Germplasm/PedigreeNode.json

```diff
@@ -92,26 +89,13 @@
                 },
                 "progeny": {
                     "description": "A list of germplasm references that are direct children of this germplasm. These represent edges in the tree, connecting to other nodes.\n<br/> The given germplasm could have a large number of progeny, across a number of different breeding methods. The 'parentType' shows \n      the type of parent this germplasm is to each of the child germplasm references.\n<br/> If the parameter 'includeProgeny' is set to false, then this array should be empty, null, or not present in the response.",
+                    "relationshipType": "one-to-many",
+					"referencedAttribute": "pedigreeNode",
                     "items": {
-                        "title": "GermplasmChild",
-                        "properties": {
-                            "progenyGermplasm": {
-                                "$ref": "Germplasm.json#/$defs/Germplasm",
-                                "description": "The ID which uniquely identifies a parent germplasm",
-                                "referencedAttribute": "parentPedigreeNodes",
-                                "relationshipType": "many-to-one"
-                            },
-                            "parentType": {
-                                "description": "The type of parent used during crossing. Accepted values for this field are 'MALE', 'FEMALE', 'SELF', 'POPULATION', and 'CLONAL'. \n\nIn a pedigree record, the 'parentType' describes each parent of a particular germplasm. \n\nIn a progeny record, the 'parentType' is used to describe how this germplasm was crossed to generate a particular progeny. \nFor example, given a record for germplasm A, having a progeny B and C. The 'parentType' field for progeny B item refers \nto the 'parentType' of A toward B. The 'parentType' field for progeny C item refers to the 'parentType' of A toward C.\nIn this way, A could be a male parent to B, but a female parent to C. ",
-                                "$ref": "ParentType.json#/$defs/ParentType"
-                            }
-                        },
-                        "required": [
-                            "progenyGermplasm",
-                            "parentType"
-                        ],
-                        "type": "object"
+                        "$ref": "../BrAPI-Germplasm/PedigreeNode.json#/$defs/Progeny",
```

Comment:
I prefer the team GermplasmChild, since it is more specific.

Reply:
Would you prefere, that we rename the whole "Progeny" Model to "GermplasmChild"?

Reply:
Yes please change :)